sympy/tensor/tests/test_tensor.py

```diff
@@ -461,6 +461,9 @@ def test_indices():
     raises(ValueError, lambda: tensor_indices(3, Lorentz))
     raises(ValueError, lambda: A(a,b,c))
 
+    A = TensorHead('A', [Lorentz])
+    assert A('a') == A(TensorIndex('a', Lorentz))
```

Comment:
Should test both paths here:
```suggestion
    assert A('a') == A(TensorIndex('a', Lorentz))
    assert A('-a') == A(TensorIndex('-a', Lorentz))
```

Reply:
Thanks for the suggestion, I will add this test. In fact, I will rewrite the test to use two indices, such that it's also possible to test the case where one index is a string, the other a `TensorIndex`.